aspnetcore/security/authentication/claims/sample6/WebRPmapClaims/Program.cs

```diff
@@ -0,0 +1,205 @@
+#define NS // FIRST SECOND NAME NS 
+#if NEVER
+#elif FIRST
+#region snippet1
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+
+var builder = WebApplication.CreateBuilder(args);
+var services = builder.Services;
```

Comment:
This is the first time I've seen us do this. I like the idea, but I think it'd be better to keep using `builder.Services` so that it's easier to drop the highlighted code into apps built with the templates.

Reply:
Good thinking.